user/views.py

```diff
@@ -18,6 +20,29 @@ def profile(username):
                            edit=(user == current_user))
 
 
+@app.route('/privacy', methods=['get'])
+@login_required
+def privacy():
+    user = User.objects.get(id=current_user.id)
+    return render_template('user/privacy.html',
+                           user=user,
+                           locations=user.locations)
+
+
+@app.route('/privacy/<string:location>/<string:date>', methods=['post'])
+@login_required
+def privacy_delete_trip(location, date):
```

Comment:
`methods` should allow delete.
URL could be improved with just a timestamp (see previous comment).